scipy/signal/tests/test_filter_design.py

```diff
@@ -4160,10 +4160,10 @@ def test_frequency_response(self):
             # and corresponding frequency.
             response_max = np.max(np.abs(response))
             freq_hz = freqs[np.argmax(np.abs(response))] / ((2 * np.pi) / fs)
-
+            
             # Check that the peak magnitude is 1 and the frequency is 1000 Hz.
```

Comment:
please be careful to avoid these whitespace changes
```suggestion

            # Check that the peak magnitude is 1 and the frequency is 1000 Hz.
```